policy/cmd/cmd/add.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cmd
```

Comment:
mmm `cmd/cmd` stutters a bit.  Is there a better way to structure the naming here?

Reply:
The use of `cmd` to contain `main.go` is consistent with the other executables in the services repo. The use of `cmd` subdirectory at the same level as `main.go` to contain individual command is the structure suggested by Cobra (https://github.com/spf13/cobra/blob/main/user_guide.md). Ergo `cmd/cmd/`.  But I agree, it _is_ a bit awkward. 

I could either 
- Deviate from Cobra suggestion and either move the commands to the  same level as `main.go`.
- Deviate from Cobra suggestion and use a different name for the commands subdirctory (what?).
- Deviate from the structure used in other places in the services repo and go with the more common golangstructure of using the executable name under `cmd/`; note  that would give us `cmd/polcli/cmd/`, which is _a little_ less awkward, by we still have the somewhat confusing duplication of "cmd".
- Do something else..?

Any preference?

Reply:
Yes you are right that it's at least consistent, which is _nearly_ always a good thing :-)

> * Do something else..?

What about introducing a new convention?

From: 
`services/${subsystem}/cmd/`

To:
`services/${subsystem}/${executable_name}/`

Reply:
Mmm, I think it's useful to have executables readily identified by being contained under `cmd/`, especially in a code base as heterogeneous as ours. So, IMO `services/${subsystem}/cmd/${executable_name}/` would be better (and aligned with golang standard practices). I could also rename the subcommands directory to `commands`. So instead of `policy/cmd/cmd/`, we could have `policy/cmd/polcli/commands/`. This does technically deviate from the Cobra suggestions, but they do look like suggestions, rather than best practices, and we still have the subcommands neatly  isolated. WDYT?

Reply:
I like it!

Reply:
Updated. I also applied the same convention to the  other executables. Please could you re-ack the pull if the new directory structure looks alright to you .